File: URI/1 - INICIANTE/Python/1051 - ImpostoDeRenda.py
salario = float(input())
if salario >=-1 and salario <= 2000.00:
    print('Isento')
elif salario > 1999.0 and salario <= 3000.00:
    print("R$ {:.1f}".format((salario-2000)*0.08))
elif salario > 2999.00 and salario <= 4500.00:
    print("R$ {:.1f}".format((1000*0.08)+(salario-3000)*0.18))
elif salario > 4499.00:
    print("R$ {:.1f}".format((1000 * 0.08) + (1500 *0.18) + (salario - 4500) * 0.28))

File: URI/1 - INICIANTE/C/1051 - ImpostoDeRenda.cpp
#include <stdio.h>

int main()
{
	float salario;
	scanf("%f", &salario);

	if (salario>= 0 && salario<= 2000.00){
		printf("Isento\n");
	}else if (salario>2000.00 && salario<=3000.00){
		printf("R$ %.2f\n", (salario - 2000.00) * 0.08);
	}

	else if (salario>3000.00 && salario<=4500.00){
		printf("R$ %.2f\n", (1000 * 0.08) + ((salario - 3000.00) * 0.18));
	}

	else if (salario>4500.00){
		printf("R$ %.2f\n", (1000 * 0.08)+(1500 * 0.18)+((salario - 4500.00) * 0.28));
	}
	
	return 0;
}
